30_news.html

```diff
@@ -120,3 +116,27 @@
     </div>
   </div>
 </div>
+
+<!-- JavaScript to Pull in last 5 Tumblr Posts -->
+<script src="{{ '/js/jquery.min.js' | prepend: site.baseurl }}"></script>
+<script src="{{ '/js/icanhazjs.min.js' | prepend: site.baseurl }}"></script>
+<script id="post" type="text/html">
+    {% raw %}<li><a href="{{ short_url }}">{{ title }}</a></li>{% endraw %}
+</script>
+<script type="text/javascript">
+  
+  $.getJSON("http://api.tumblr.com/v2/blog/playtestplay.tumblr.com/posts?limit=5&api_key=phzFQ2Zknq0Itb86tEqJ0o2Z2wqgObHLrOx6wUXGobPgSz7vL7&callback=?", function (data) {
```

Comment:
I'm a little worried about revealing an API key in github (even if this is a dummy key).  I know bots search for keys in code all the time.  After an annoyingly lengthy amount of google, I was able to find a public JSON URL for this.  http://blog.mayday.us/api/read/json   Can we use that instead of the API request?  Thanks!